tests/copilot_helper/test_command_bootstrap.py

```diff
@@ -110,6 +111,12 @@ def test_make_config(tmp_path):
         in result.output
     )
 
+    assert (tmp_path / ".copilot-version-file").exists()
+
+    with open(str(tmp_path / ".copilot-version-file")) as copilot_version_file:
+        copilot_version = ".".join([str(num) for num in get_copilot_versions()])
```

Comment:
May be better to mock out the get_copilot_versions() to return a specific list of ints and compare against an expected version number rather than repeating the actual code logic here.